### PR TITLE
Adding .zenodo.json

### PR DESCRIPTION
**Pull request by gregstarr**
Copied and edited https://github.com/aburrell/aacgmv2/blob/main/.zenodo.json to address #34

### CODE REVIEW

File: .zenodo.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  "creators": [
+    {
+      "orcid": "0000-0001-8875-9326", 
+      "affiliation": "Naval Research Laboratory", 
+      "name": "Angeline G. Burrell"
+    }, 
+    {
+      "orcid": "0000-0002-8043-0953", 
+      "name": "Christer van der Meeren"
+    }
```
Comment (aburrell) [inline .zenodo.json:32]: We need to add the rest of the authors from the Authors.md file here.  The order should be:
1) Christer
2) Karl
3) Angeline
4) Greg
5) Ashton
6) Achim

Comment (gregstarr) [reply .zenodo.json:32]: Ok wasn't sure, thanks for the feedback

Comment (gregstarr) [reply .zenodo.json]: @sputnik-a is this you?

https://orcid.org/0000-0003-4227-8687

Comment (gregstarr) [reply .zenodo.json]: Nope that's not him. I don't think he has orcid

Comment (sputnik-a) [reply .zenodo.json]: No, that's not me. Here is my Orc-ID: https://orcid.org/0000-0001-7955-4441

**Review by aburrell — APPROVED** [review]

**Pull request merged**